### DISCUSSION TITLE
Fortran runtime error: Index '6733' of dimension 1 of array 'lchunks' above upper bound of 6732

### DISCUSSION THREAD

**Discussion by rytam2** [E3SM-Project/E3SM#6122] [E3SM model help]
Hello! I am trying to run an AMIP-style simulation with F2010 compset at 1degree resolution on a local machine. I am trying to run the case with the SST boundary condition set to the [AMIP-1-1-9 SSTs](https://handle-esgf.dkrz.de/lp/21.14100/cba3f014-387c-4120-ba1a-14200768c1d3) in the period 1999-2022 (full 24 years), while holding sea ice the same as the F2010 compset default ( `sst_ice_CMIP6_DECK_E3SM_1x1_2010_clim_c20190821.nc`) for the 24 years. After creating the case, I have set `RUN_STARTDATE=1999-1-16`, `SSTICE_YEAR_START=1999`, `SSTICE_YEAR_END=2022` and `SSTICE_DATA_FILENAME` to the file with both SST and SIC as described above, while keeping SSTICE_YEAR_ALIGN=1. 

The case built successfully, but the case fails after ~13 seconds into the run with the following error. May I know what might be the possible cause of the error?  
```
At line 1015 of file /data/keeling/a/rytam2/E3SM2.0/E3SM/components/eam/src/physics/cam/phys_grid.F90
Fortran runtime error: Index '6733' of dimension 1 of array 'lchunks' above upper bound of 6732
```

When creating the 24-year boundary condition, I have interpolated the SSTs from AMIP-1-1-9 to those in F2010 default and changed the time coordinates and the time_bnds variable to those in AMIP-1-1-9 (from float64 to datetime64 format). I have tried submitting the case with both types of time coordinates, but they still fail with the same error. 

On top of possible issues from the boundary condition file, I am unsure if I am missing certain settings (i.e. do I need to change more variables with `./xmlchange`?) when running a case that has boundary conditions longer than 1 year, as the F2010 SSTICE boundary condition only has data for 12 months, while I am loading 288 months. 

Any advice would be much appreciated!

**Comment by jonbob** [comment]
Hi Rachel,

My name is Jon Wolfe and I work on E3SM at LANL.

When you say 1-degree, exactly what resolution are you referring to in E3SM? Can you tell me what command you've run to create your case?

Thanks - Jon

**Reply by rytam2** [reply]
Hello Jon! The resolution is ne30pg2_EC30to60E2r2. When creating the case, I simply call the following: 
```
./create_newcase --case *casename* --compset F2010 --res ne30pg2_EC30to60E2r2
```

Cheers,
Rachel

**Comment by rljacob** [comment]
What does the output of "./pelayout" show in your case directory?

**Reply by rytam2** [reply]
Hello! It looks like the following: 
```
Comp  NTASKS  NTHRDS  ROOTPE PSTRIDE
CPL :     48/    16;      0      1
ATM :     48/    16;      0      1
LND :     48/    16;      0      1
ICE :     48/    16;      0      1
OCN :     48/    16;      0      1
ROF :     48/    16;      0      1
GLC :     48/    16;      0      1
WAV :     48/    16;      0      1
IAC :      1/    16;      0      1
ESP :      1/    16;      0      1
```

**Reply by mt5555** [reply]
NTHRDS=16 is much too high (unless you have an obsolete KNL system?)  Probably change that to 1, and increase NTASKS to the number of cores per node x number of nodes you want to run on.

**Comment by jonbob** [comment]
Thanks Rachel,

Which atm file did you manipulate? And exactly which fields? I was curious about the specific resolution because the pg2 meshes have different sized fields, some on the physgrid and some on the dynamics. In the older meshes those were the same, and the error in phys_grid code makes me suspicious that the new data might have the incorrect number of columns....

Jon
